### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by DirtyRacer1337**

### CODE REVIEW

File: namer/metadataapi.py [hunk 1]
```diff
@@ -188,7 +188,8 @@ def __match_weight(result: ComparisonResult) -> float:
     value = 0.00
     if result.phash_distance is not None:
```
Comment (4c0d3r) [inline namer/metadataapi.py:189]: notice we're already in a phash block here..... and if the value can't reach 1000 for a perfect phash match, these will always be behind a name match, and possibly name failures.....

Comment (DirtyRacer1337) [reply namer/metadataapi.py:189]: okay, maybe this 70e821f

Comment (4c0d3r) [reply namer/metadataapi.py:189]: sure, worth a shot

File: namer/metadataapi.py [hunk 2]
```diff
@@ -188,7 +188,7 @@ def __match_weight(result: ComparisonResult) -> float:
...
         logger.debug("Phash match with '{} - {} - {}'", result.looked_up.site, result.looked_up.date, result.looked_up.name)
-        value = 1000.00 - result.phash_distance * 100
+        value += 1000 - result.phash_distance * 125
         if result.site_match:
```
Comment (4c0d3r) [inline namer/metadataapi.py:192]: maybe move these other if blocks out of the phash block and switch name match below to be (900 + the name percent) that way a phash with distance of 2 has a fighting chance to be first?

Comment (4c0d3r) [reply namer/metadataapi.py:192]: just thinking outlaid about how to balance all the permutations.

**Pull request merged**